### ISSUE TITLE
After adding inner margins to CodeArea, VirtualizedScrollPanel displays abnormally

### ISSUE THREAD

**Issue by ljy1058318852** [FXMisc/RichTextFX#1194]
VirtualizedScrollPanel contains CodeArea. There is no problem with the default scroll bar

But when CodeArea sets the padding (VirtualizedScrollPanel setting padding is invalid), the scroll bar only appears when the internal string of CodeArea exceeds several digits.

`
<VirtualizedScrollPane minHeight="300" minWidth="300"  style="-fx-background-color: white">
    <content>
        <CodeArea  style="-fx-background-color: red;-fx-padding:20"/>
    </content>
</VirtualizedScrollPane>
`

> richtextfx v0.10.8
> java 11

What should I do
Do you need to upgrade richtextfx version

**Comment by Jugen** [comment]
Have found the bug in [Flowless ](https://github.com/FXMisc/Flowless) and will submit a PR to fix ....

**Comment by ljy1058318852** [comment]
Thank you.

**Comment by ljy1058318852** [comment]
After you submit the PR, do I need to download new git source code for association in my project instead of using MVN to automatically download it

**Comment by Jugen** [comment]
Yes, if you want to use it immediately. However I'm probably going to publish a new release of Flowless today or tomorrow, and then RichTextFX next week.

**Comment by ljy1058318852** [comment]
Thank you very much. I have chosen to wait for the release to facilitate component version management. Thank you very much

**Comment by Jugen** [comment]
In case you haven't seen `0.11.2` is now available on Maven
